addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -16,9 +16,9 @@ registerModel({
     identifyingFields: ['messaging'],
     lifecycleHooks: {
         _willDelete() {
-            if (this.env.services['legacy_bus_service']) {
-                this.env.services['legacy_bus_service'].off('notification');
-                this.env.services['legacy_bus_service'].stopPolling();
+            if (this.env.services['bus_service']) {
+                this.env.services['bus_service'].offNotification();
+                this.env.services['bus_service'].stopBus();
```

Comment:
This is probably wrong, if other parts of the code are using the bus there is no reason that messaging handler should be responsible of stopping it.